lib/galaxy/datatypes/sequence.py

```diff
@@ -693,8 +694,7 @@ def process_split_file(data):
         else:
             commands = Sequence.get_split_commands_sequential(is_gzip(input_name), input_name, output_name, start_sequence, sequence_count)
         for cmd in commands:
-            if 0 != os.system(cmd):
-                raise Exception("Executing '%s' failed" % cmd)
+            subprocess.check_call(cmd, shell=True)
```

Comment:
remove shell=True?

Reply:
I'm not going to do this one now. There are multiple + varying numbers of pipes in the commands passed to it, this is non-trivial to refactor currently.